src/serac/physics/thermal_solid.hpp

```diff
@@ -36,6 +36,16 @@ class ThermalSolid : public BasePhysics {
   ThermalSolid(int order, std::shared_ptr<mfem::ParMesh> mesh, const ThermalConduction::SolverOptions& therm_options,
                const NonlinearSolid::SolverOptions& solid_options);
 
+  /**
+   * @brief Construct a new Thermal Solid object from input deck options
```

Comment:
Trying to move away from "input deck", please use "input file".  I know I keep doing it myself.

Reply:
BUT I'M OLDDDDD.

Reply:
=( it's so hard....